Application/PlaybackEventController.m

```diff
@@ -172,6 +172,32 @@ - (void)performPlaybackDidBeginActions:(PlaylistEntry *)pe
                     content.sound = nil;
                     content.categoryIdentifier = @"play";
                     
+                    if ([defaults boolForKey:@"notifications.show-album-art"] && [pe albumArtInternal]) {
+                        NSError *error = nil;
+                        NSFileManager *fileManager = [NSFileManager defaultManager];
+                        NSString *tmpSubFolderName = [NSProcessInfo.processInfo globallyUniqueString];
+                        NSURL *tmpSubFolderURL = [[NSURL fileURLWithPath:NSTemporaryDirectory()]
+                                URLByAppendingPathComponent:tmpSubFolderName isDirectory:true];
+                        if ([fileManager createDirectoryAtPath:[tmpSubFolderURL path]
+                                   withIntermediateDirectories:true
+                                                    attributes:nil
+                                                         error:&error]) {
+                            NSURL *fileURL = [tmpSubFolderURL URLByAppendingPathComponent:@"art.png"];
```

Comment:
Nope. And I can't suggest using a unique name, either, because then it's on us to wipe out the old files when placing a new file. Maybe using a specific temp file name pattern, and wiping them all on startup, exit, and before placing a new file, but still using a unique name, so that old requests pull the missing file, while new are guaranteed to always get the newly generated file. Or something.

Reply:
Oh, it already produces a unique name. Huh, sorry I didn't notice that sooner.